LOAN/LoanContracts/LoanHMDA.rdf

```diff
@@ -89,21 +107,53 @@ Copyright (c) 2015-2018 Object Management Group, Inc.</sm:copyright>
 		<fibo-fnd-utl-av:adaptedFrom>the Revised HMDA regulatory text.</fibo-fnd-utl-av:adaptedFrom>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-loan-ln-hmda;HMDA_Disposition">
+	<owl:Class rdf:about="&fibo-loan-ln-hmda;HMDA-Disposition">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-cls;Classifier"/>
 		<rdfs:label>HMDA disposition</rdfs:label>
 		<skos:definition>a type of action taken regarding an application for a HMDA covered loan</skos:definition>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-loan-ln-hmda;HMDA_PreApprovalRequest">
-		<rdfs:subClassOf rdf:resource="&fibo-loan-ln-ln;PreApprovalRequest"/>
+	<owl:NamedIndividual rdf:about="&fibo-loan-ln-hmda;HMDA-Disposition-ApplicationApprovedButNotAccepted">
+		<rdf:type rdf:resource="&fibo-loan-ln-hmda;HMDA-Disposition"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-ln-hmda;HMDA-Disposition-ApplicationDenied">
+		<rdf:type rdf:resource="&fibo-loan-ln-hmda;HMDA-Disposition"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-ln-hmda;HMDA-Disposition-ApplicationWithdrawn">
+		<rdf:type rdf:resource="&fibo-loan-ln-hmda;HMDA-Disposition"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-ln-hmda;HMDA-Disposition-FileClosedForIncompleteness">
+		<rdf:type rdf:resource="&fibo-loan-ln-hmda;HMDA-Disposition"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-ln-hmda;HMDA-Disposition-LoanOriginated">
+		<rdf:type rdf:resource="&fibo-loan-ln-hmda;HMDA-Disposition"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-ln-hmda;HMDA-Disposition-LoanPurchasedByYourInstitution">
```

Comment:
Yes, of course they do.  Again, I moved them out of LoanCore because they were HMDA-specific, but did not do anything with the content aside from eliminating "_" at the front of individual names and using "-" instead of "_" as we've done with other generated content.  

The LOAN working group may have HMDA SMEs once that kicks off, which would be better than any attempted guesses I might have by looking at the regulation but without the background knowledge that the folks that contributed this content had.